integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/cache/CarbonShowCacheCommand.scala

```diff
@@ -74,6 +75,7 @@ case class CarbonShowCacheCommand(tableIdentifier: Option[TableIdentifier],
         .listAllTables(sparkSession).filter {
         carbonTable =>
           carbonTable.getDatabaseName.equalsIgnoreCase(currentDatabase) &&
+          isValidTable(carbonTable, sparkSession) &&
```

Comment:
It's better use 'Checker.validateTableExists(databaseNameOp, tableName.get, sparkSession)'.

Reply:
I don't need to throw an error if the table does not exist, I just need a true or false value.

Reply:
Got it.